schemas/panels/time-series/time-series.cue

```diff
@@ -21,12 +21,18 @@ import (
 	position?: "bottom" | "right"
 }
 
+#visual: {
+	line_width?:   number
+	point_radius?: number
```

Comment:
does these numbers have a range ? Like they cannot be negative and they cannot be superior than 100 for example ?

Reply:
just asking, because I think it's possible to put some constraints about a range in Cue.
@AntoineThebaud relying on you as our Cue expert :)

Reply:
Yep you can write sth like
```cue
line_width?: number & >0 & <100
```
or more simply
```cue
line_width?: >0 & <100
```

and to avoid repetition:
```cue
#short_positive: >=0 & <=100
line_width?: #short_positive
point_radius?: #short_positive
```
(btw if we're going to have the same constraint in other cases it could make sense to move it to the common lib).

Reply:
That's really cool, yeah I had no idea how to do that in Cue, but will update time-series.cue based on props I set in MUI Slider